src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
+
+    public Ball() {
```

Comment:
There is no need to create empty constructor, as there is no need for `Ball` to exist without defined number and color. This constructor should be removed, and only the constructor taking color and number as arguments should be used.